modules/user.nix

```diff
@@ -40,7 +40,7 @@ in
 
       shell = mkOption {
         type = types.path;
-        readOnly = true;
+        default = "/bin/sh";
```

Comment:
I think I've written this before, but I don't see this comment, so sorry if it'll be a duplicate.

Why `/bin/sh`? Isn't it a user experience regression? Can it somehow default to `${pkgs.bash}/bin/bash`?

Reply:
Yes of course, you are right, I'll update this